newindex.html

```diff
@@ -0,0 +1 @@
+‘New file’
```

Comment:
Please enclose this within the html tags

Reply:
ok